tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/HazelcastSessionDistributionTests.java

```diff
@@ -0,0 +1,263 @@
+                    assertThat(response.getContentAsString(), containsString("SESSION READ CHOCOLATE THE BEST:FRENCH"));
+                }
+            }
+
+        }
+    }
+
+    @Disabled("not working see https://github.com/hazelcast/hazelcast/issues/18508")
+    /**
+     * This test simulate Hazelcast instance within Jetty a cluster member with an external Hazelcast instance
+     */
+    public void testHazelcastRemoteAndPartOfCluster() throws Exception
+    {
+
+        Map<String, String> env = new HashMap<>();
+        // -Dhazelcast.local.publicAddress=127.0.0.1:5701
+        env.put("JAVA_OPTS", "-Dhazelcast.config=/opt/hazelcast/config_ext/hazelcast.xml");
+        try (GenericContainer hazelcast =
+                 new GenericContainer("hazelcast/hazelcast:" + System.getProperty("hazelcast.version", "3.12.6"))
+                     .withExposedPorts(5701, 5705)
+                     .withEnv(env)
+                     .waitingFor(Wait.forLogMessage(".*is STARTED.*", 1))
+                     //.withNetworkMode("host")
+                     //.waitingFor(Wait.forListeningPort())
+                     .withClasspathResourceMapping("hazelcast-server.xml",
+                                                   "/opt/hazelcast/config_ext/hazelcast.xml",
+                                                   BindMode.READ_ONLY)
+                     .withLogConsumer(new Slf4jLogConsumer(HAZELCAST_LOG)))
+        {
+            hazelcast.start();
+            String hazelcastHost = InetAddress.getByName(hazelcast.getContainerIpAddress()).getHostAddress(); // hazelcast.getContainerIpAddress();
+            int hazelcastPort = hazelcast.getMappedPort(5701);
+//            int hazelcastMultiCastPort = hazelcast.getMappedPort(54327);
+
+            LOGGER.info("hazelcast started on {}:{}", hazelcastHost, hazelcastPort);
+
+            Map<String, String> tokenValues = new HashMap<>();
+            tokenValues.put("hazelcast_ip", hazelcastHost);
+            tokenValues.put("hazelcast_port", Integer.toString(hazelcastPort));
+//            tokenValues.put("hazelcast_multicast_port", Integer.toString(hazelcastMultiCastPort));
+            Path hazelcastJettyPath = Paths.get("target/hazelcast-jetty.xml");
+            transformFileWithHostAndPort(Paths.get("src/test/resources/hazelcast-jetty.xml"),
+                                         hazelcastJettyPath,
+                                         tokenValues);
+
+            String jettyVersion = System.getProperty("jettyVersion");
+            DistributionTester distribution = DistributionTester.Builder.newInstance()
+                .jettyVersion(jettyVersion)
+                .mavenLocalRepository(System.getProperty("mavenRepoPath"))
+                .build();
+
+            String[] args1 = {
+                "--create-startd",
+                "--approve-all-licenses",
+                "--add-to-start=resources,server,http,webapp,deploy`,jmx,servlet,servlets,session-store-hazelcast-remote"
```

Comment:
There's a back quote char after `deploy`.

Reply:
good catch. arghh and CI cannot see it as it's not executed :)